src/models/p2pkh.ts

```diff
@@ -44,7 +44,7 @@ class P2PKH {
    * @memberof P2PKH
    * @inner
    */
-  getType(): string {
+  getType(): 'p2pkh' {
```

Comment:
It is a string and the value is always 'p2pkh'.
This is so that we can have a `address.getType(): 'p2pkh'|'p2sh'|'data'` instead of `address.getType(): string`.
This effectively works as a string enum for typing and allows for better type checking.

We could change it to an actual enum, but there were some typing issues, so I went with this simpler approach